### PR TITLE
Syntax lookup: let \""

### PR DESCRIPTION
**Pull request by kevanstannard**
Suggestion for #162

### CODE REVIEW

**Comment by bobzhang** [comment]
This feature should not be exposed to end users. It may break lots of compiler invariants

**Comment by kevanstannard** [comment]
Thanks @bobzhang 

@ryyppy what do you think? Should we remove the content here:

https://rescript-lang.org/docs/manual/latest/use-illegal-identifier-names

**Comment by ryyppy** [comment]
Yeah the way this PR phrases it makes it look like a feature that users can safely use without any consequences. It is supposed to be an escape hatch only feature. 

Will discuss this internally. I believe this feature should be documented (but phrased differently), otherwise it might be abused. The goal is to document all language constructs available in the language.

I will adapt your texts later on, just keep the PR open.

**Comment by kevanstannard** [comment]
Hi @ryyppy @bobzhang anything I can do to help resolve this one? I can have a go at adapting the text, or whatever else you have in mind. Happy to be guided by you.

File: misc_docs/syntax/language_let_invalid.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+id: "let-binding-invalid"
+keywords: ["let", "variable"]
+name: "let \\\"\""
+summary: "This is the `let binding` for invalid identifier names."
+category: "languageconstructs"
+---
+
+`let \""` is used to bind values to names that are invalid in ReScript, including capitalized names, names with special characters, and reserved keywords.
```
Comment (ryyppy) [inline misc_docs/syntax/language_let_invalid.mdx:9]:
```suggestion
`let \""` is used to bind values to names that are invalid in ReScript, including capitalized names, names with special characters, and reserved keywords.

**Warning: ** This escape hatch is almost exclusively used for interop with JavaScript. If you abuse this, many of the compiler guarantees will go away.
```

**Pull request merged**